packages/orchestration/src/fetched-chain-info.js

```diff
@@ -9,6 +9,7 @@ export default /** @type {const} } */ ({
       },
     ],
     icqEnabled: false,
+    pfmEnabled: true,
```

Comment:
Putting stuff that does not come from the chain registry into fetched-chain-info doesn't seem right to me.

Whatever consumes fetched-chain-info should mix in `PFM_ENABLED`, IMO.